### DISCUSSION TITLE
Ability to compare data in multiple tables and "choose" which rows to migrate from stage to prod

### DISCUSSION THREAD

**Discussion by pmonco** [dbeaver/dbeaver#36182] [Q&A]
I can create the compare between two tables and save as a task, but I would love to be able chain a bunch of compare tasks together, have each task export the data from the source and target tables, along with the respective change statement (update, insert, delete), so that I can review en masse and promote select rows. 

Our database is two columns, the id (pk) and a jsonb column, so reviewing the data in the compare viewer isn't great.

**Comment by E1izabeth** [comment]
Have you tried Data Compare feature?
https://dbeaver.com/docs/dbeaver/Data-compare/

**Reply by pmonco** [reply]
Hey E1izabeth, appreciate the response. I have used data compare, but it doesn't seem like I can automate the export of the changes (inserts and updates), or pick which rows I update or revert. I totally understand that I might be asking for something quite "extra" to occur alongside or after Data Compare.

**Reply by E1izabeth** [reply]
Could you create a feature request about that?
https://github.com/dbeaver/dbeaver/issues/new?assignees=&labels=feature+request%2C+wait+for+review&projects=&template=feature_request.md&title=
